cvxpy 1.1.7 no longer compatible with numpy<1.20?

I use python 3.7.9 with cvxpy 1.1.7 and numpy 1.18.1.
However as of Sunday we get an error message that the module is compiled with a newer numpy version. Before the weekend everything was still working.

What happened on sunday 31 january that this throws an error now?
Issue is resolved by upgrading numpy to 1.20, however isnt cvxpy supposed to support numpy>1.15? I want to stay on 1.18.1 still.

import cvxpy as cp
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/__init__.py:18: in <module>
    from cvxpy.atoms import *
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/atoms/__init__.py:20: in <module>
    from cvxpy.atoms.geo_mean import geo_mean
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/atoms/geo_mean.py:20: in <module>
    from cvxpy.utilities.power_tools import (fracify, decompose, approx_error, lower_bound,
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/utilities/power_tools.py:18: in <module>
    from cvxpy.atoms.affine.reshape import reshape
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/atoms/affine/reshape.py:18: in <module>
    from cvxpy.atoms.affine.hstack import hstack
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/atoms/affine/hstack.py:18: in <module>
    from cvxpy.atoms.affine.affine_atom import AffAtom
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/atoms/affine/affine_atom.py:22: in <module>
    from cvxpy.cvxcore.python import canonInterface
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/cvxpy/cvxcore/python/__init__.py:3: in <module>
    import _cvxcore
E   ImportError: numpy.core.multiarray failed to import
------------------------------- Captured stderr --------------------------------
RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["numpy >= 1.15", "scipy >= 1.1.0", "setuptools>=40.8.0", "wheel"]
+requires = ["oldest-supported-numpy", "scipy >= 1.1.0", "setuptools>=40.8.0", "wheel"]

